commands/cluster-shards.md

```diff
@@ -0,0 +1,150 @@
+`CLUSTER SHARDS` returns details about the shards of the cluster.
+A shard is defined as a collection of nodes that serve the same set of slots and that replicate from each other.
+A shard may only have a single master at a given time, but may have multiple replicas.
```

Comment:
There is also the case where a shard has no replica but just primary (master). I think it is worth calling out explicitly as well.